airflow/config_templates/config.yml

```
@@ -361,6 +361,15 @@
       type: string
       example: ~
       default: "downstream"
+    - name: default_execution_timeout
+      description: |
+        The default execution_timeout value for the operators. Expected positive integer value to
+        be passed into timedelta as seconds. The missing and the non-positive values are considered as None,
+        means that the operators are never timed out.
+      version_added: 2.3.1
```

```suggestion
      version_added: 2.3.0
```

We can target 2.3.0 for this one

---

done ✅